cmd_plus.go

```diff
@@ -52,14 +52,22 @@ func (c *CmdPlus) Kill() error {
 	return nil
 }
 
+// GetOutput returns the output thus far
+func (c *CmdPlus) GetOutput() string {
+	c.mutex.Lock()
+	result := c.output
+	c.mutex.Unlock()
```

Comment:
Will update to using defer. 

> Also, is this mutex used elsewhere? Might make sense to use our own mutex here

The mutex is used to prevent race conditions in reading / writing c.output and reading / writing c.outputChannels.